<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Dimension">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Of first closest drillholes</string>
   </property>
  </widget>
  <widget class="QLabel" name="MaximumL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="Interpolation_L">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute Distance To</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComputeDistanceTo">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>First Closest Hole</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>First N Closest Holes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All Drillholes</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="NbClosestDrillholes">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseButton">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MinimumDistance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Distances</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
